index.html

```diff
@@ -9,111 +9,271 @@
+            <li>Tag 4</li>
+          </ul>
+        </article>
+
+        <article class="card" data-js="card">
+          <h2 class="card__h2">Question:</h2>
+          <button
+            class="card__bookmark card__bookmark--clicked card__bookmark--style"
+            data-js="bookmark-clicked"
+          >
+            <img
+              src="../../Images/bookmark-new-48x48.svg"
+              alt=""
+              width="40px"
+              height="40px"
+            />
+          </button>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua. At vero eos et accusam et justo duo dolores
+            et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est
+            Lorem ipsum dolor sit amet.
+          </p>
+          <details>
+            <summary data-js="card__show-answer-button">Show Answers</summary>
+            <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</p>
+          </details>
+          <ul role="list" class="card__tag">
+            <li>Tag 1</li>
+            <li>Tag 2</li>
+            <li>Tag 3</li>
+            <li>Tag 4</li>
+          </ul>
+        </article>
+      </section>
+    </main>
 
-    <main class="app-grid__scrollarea">
-      <article class="card">
-        <h2 class="card__h2">Question:</h2>
-        <button
-          class="card__bookmark card__bookmark--style card__bookmark--clicked"
-          data-js="bookmark-clicked"
+    <main class="app-grid hidden" data-page="create">
+      <h3 class="app-grid__header">Create</h3>
+      <section class="app-grid__scrollarea">
+        <form
+          aria-labelledby="create-heading"
+          autocomplete="off"
+          class="survey"
         >
+          <label for="Question" class="survey-label" data-form="form"
+            >Add your Question:
+            <output class="survey__output--question" data-js="output"
+              >200</output
+            >
+            <textarea
+              name=""
+              id="question-field"
+              maxlength="200"
+              required
+              class="survey__textarea"
+              data-js="surveyTextarea"
+            ></textarea>
+          </label>
+          <label for="Answer" class="survey-label" data-form="form"
+            >Add your Answer:
+            <output class="survey__output--answer" data-js="output">200</output>
+            <textarea
+              name=""
+              id="answer-field"
+              maxlength="200"
+              required
+              class="survey__textarea"
+              data-js="surveyTextarea"
+            ></textarea>
+          </label>
+          <label for="tags-field" class="survey-label" data-form="form"
+            >Add your Tags:
+            <output class="survey__output--tags" data-js="output-tags"
+              >100</output
```

Comment:
I think it should be 200, as your maxlength and also down in the js file you wrote 200 😊

Reply:
200 is for my textareas. The 100 charaktars are for my tags. I didn´t want there to be as long of a tag as there is a question or answer. I put those in a seperate js file.